styles.css

```diff
@@ -80,8 +70,7 @@ body {
     }
 
     .a {
-        grid-area: a;
-
+        grid-area: a
     }
```

Comment:
Ook deze heb je hier boven al een naam gegeven dus hoeft je niet nogmaals in je code te zetten. Als je hem buiten je container plaatst onder je desktop en mobiele container pakt hij hem voor beide. Nu is je code een beetje rommelig omdat dingen er dubbel instaan.